src/ipips/ipip-0002.md

```diff
@@ -0,0 +1,87 @@
+---
+date: 2022-06-15
+maturity: proposal
```

Comment:
:thought_balloon: having rattified and shipped proposals marked as "proposal" feels weird.

@hacdias  @darobin  thoughts on adding second status for IPIPs named  'ratified' and have analogous watermark?
That was it would be visually clear when a proposal vs ratified spec change description is displayed.

Reply:
Perhaps we can use a completely different tag altogether. I had to add `if`s in the spec-generator since this maturity is handled differently from the remaining ones. What about: `ipip: proposal` and `ipip: ratified`? I

Reply:
I added the aforementioned options.